### ISSUE TITLE
load model error

### ISSUE THREAD

**Issue by Observer007** [fizyr/keras-maskrcnn#89]
When I load the pre-trained model in jupyter notebook, an error comes out.
ValueError                                Traceback (most recent call last)
/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/array_ops.py in zeros(shape, dtype, name)
   1804         # to prevent serialized GraphDefs from becoming too large.
-> 1805         output = _constant_if_small(zero, shape, dtype, name)
   1806         if output is not None:

/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/array_ops.py in _constant_if_small(value, shape, dtype, name)
   1761   try:
-> 1762     if np.prod(shape) < 1000:
   1763       return constant(value, shape=shape, dtype=dtype, name=name)

/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/gen_math_ops.py in less(x, y, name)
   4621     _, _, _op = _op_def_lib._apply_op_helper(
-> 4622         "Less", x=x, y=y, name=name)
   4623   except (TypeError, ValueError):

/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/op_def_library.py in _apply_op_helper(self, op_type_name, name, **keywords)
    787                          input_types=input_types, attrs=attr_protos,
--> 788                          op_def=op_def)
    789       return output_structure, op_def.is_stateful, op

/usr/local/lib/python3.5/dist-packages/tensorflow/python/util/deprecation.py in new_func(*args, **kwargs)
    506                 instructions)
--> 507       return func(*args, **kwargs)
    508 

/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/ops.py in create_op(***failed resolving arguments***)
   3299           original_op=self._default_original_op,
-> 3300           op_def=op_def)
   3301       self._create_op_helper(ret, compute_device=compute_device)

/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/ops.py in __init__(self, node_def, g, inputs, output_types, control_inputs, input_types, original_op, op_def)
   1837     if not c_op:
-> 1838       self._control_flow_post_processing()
   1839 

/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/ops.py in _control_flow_post_processing(self)
   1846     for input_tensor in self.inputs:
-> 1847       control_flow_util.CheckInputFromValidContext(self, input_tensor.op)
   1848     if self._control_flow_context is not None:

/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/control_flow_util.py in CheckInputFromValidContext(op, input_op)
    334     logging.info(log_msg)
--> 335     raise ValueError(error_msg + " See info log for more details.")

ValueError: Cannot use 'roi_align_1/map/while/strided_slice_20' as input to 'roi_align_1/map/while/zeros/Less' because 'roi_align_1/map/while/strided_slice_20' is in a while loop. See info log for more details.

**Comment by Observer007** [comment]
My tf version is 1.13.1, keras-rcnn is 0.2.2, keras 2.3.1 and keras-retinanet is 0.5.1

**Comment by hgaiser** [comment]
I'm working to resolve this issue. For now you should be able to run keras 2.2.5 without an issue.

**Comment by Wemmy** [comment]
I was using keras 2.2.5, but it shows:

You are using keras version 2.2.5. The minimum required version is 2.3.0.

Still doesn't work with version 2.2.5

**Comment by hgaiser** [comment]
Yeah you will also need the exact version 0.5.1. You can do `git checkout 0.5.1` or install the latest stable version using `pip install --user keras-retinanet`.

**Comment by thebeyonder001** [comment]
The error still persists
Any solution?

**Comment by paulaceccon** [comment]
Same problem.
